components/inlinealert/stories/template.js

```diff
@@ -57,17 +83,52 @@ export const Template = ({
 			class=${classMap({
 				[rootClass]: true,
 				[`${rootClass}--${variant}`]: typeof variant !== "undefined",
+				[`${rootClass}--${treatment}`]: treatment !== "border",
 				...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 			})}
 			style=${styleMap(customStyles)}
 
 		>
-			<div class="${rootClass}-header">
-				${headerText}
-				${iconMarkup}
-			</div>
+			${titleMarkup}
 			<div class="${rootClass}-content">${text}</div>
+			${linkMarkup}
 			${closableMarkup}
 		</div>
 	`;
 };
+
+/**
+ * Convert provided string to title case
+ */
+const toTitleCase = (string) => string.replace(/\w\S*/g, text => text.charAt(0).toUpperCase() + text.substring(1).toLowerCase());
+
+/**
+ * Set the appropriate treatment header text
+ */
+const setTreatmentHeaderText = (variant, treatment) => `${toTitleCase(variant)} variant with ${treatment !== "border" ? treatment : "outline"} fill`;
+
+export const AlertsWithStyleOptions = ({
+	...args
+}, context = {}) => Container({
+	withBorder: false,
+	direction: "row",
+	wrapperStyles: {
+		columnGap: "12px",
+	},
+	content: html`
+		${Template({
+			...args,
+			headerText: setTreatmentHeaderText(args.variant, "border"),
+		}, context)}
+		${Template({
+			...args,
+			treatment: "subtle",
+			headerText: setTreatmentHeaderText(args.variant, "subtle"),
+		}, context)}
+		${Template({
+			...args,
+			treatment: "bold",
+			headerText: setTreatmentHeaderText(args.variant, "bold"),
+		}, context)}
+	`,
```

Comment:
```suggestion
	content: [
		Template({
			...args,
			headerText: setTreatmentHeaderText(args.variant, "border"),
		}, context),
		Template({
			...args,
			treatment: "subtle",
			headerText: setTreatmentHeaderText(args.variant, "subtle"),
		}, context),
		Template({
			...args,
			treatment: "bold",
			headerText: setTreatmentHeaderText(args.variant, "bold"),
		}, context),
	],
```

Might be simpler to use the content array technique instead of having to form the combination yourself here.